src/main/java/com/thanksd/server/service/MemberService.java

```diff
@@ -61,4 +64,26 @@ public OAuthMemberSignUpResponse signUpByOAuthMember(OAuthMemberSignUpRequest re
         member.registerOAuthMember(request.getEmail());
         return new OAuthMemberSignUpResponse(member.getId());
     }
+
+    @Transactional
+    public DiaryIdResponse deleteDiary(Long memberId, Long diaryId) {
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new NotFoundMemberException());
```

Comment:
```suggestion
                .orElseThrow(NotFoundMemberException::new);
```


코드 보면 `Lambda can be replaced with method reference` 이게 뜨는데 예외 던지는 코드를 이렇게 바꿀 수 있을 거 같아요!